internal/cloud/realm/user_ext_test.go

```diff
@@ -99,17 +100,22 @@ func TestRealmUsers(t *testing.T) {
 
 			t.Run("And disable users", func(t *testing.T) {
 				assert.Nil(t, client.DisableUser(groupID, testApp.ID, email1.ID))
-				assert.Nil(t, client.DisableUser(groupID, testApp.ID, email3.ID))
-			})
-
-			t.Run("And find all disabled users", func(t *testing.T) {
-				users, err := client.FindUsers(groupID, testApp.ID, realm.UserFilter{State: realm.UserStateDisabled})
-				assert.Nil(t, err)
+				users1, users1Err := client.FindUsers(groupID, testApp.ID, realm.UserFilter{IDs: []string{email1.ID}})
+				assert.Nil(t, users1Err)
+				email1Disabled := users1[0]
+				assert.True(t, email1Disabled.Disabled, fmt.Sprintf("expected %s to be disabled", email1Disabled.Data["email"]))
```

Comment:
[optional] this test data doesn't live around too long, so the id values aren't too too helpful here.  feel free to leave as is, but having "expected user to be disabled" could suffice